src/Microsoft.Azure.WebJobs.Extensions.Sql.csproj

```
@@ -46,4 +46,8 @@
     <Copy SourceFiles="@(_Packages)" DestinationFolder="../local-packages" />
     <Message Text="Copied sql .nupkg to local-packages" Importance="high" />
   </Target>
+  <Target Name="RemoveLocalPackages" BeforeTargets="Build" Condition="'$(Configuration)' == 'Debug'">
```

Making the name match what it's actually doing and removing the debug condition (it should run regardless, there's no reason to only do it for debug builds)

```suggestion
  <Target Name="RemoveCachedExtensionPackage" BeforeTargets="Build">
```